Src/System.Linq.Dynamic.Tests/EntitiesTests.cs

```diff
@@ -189,10 +189,10 @@ public void Entities_Where_ByDateTime_AsTypeAccess()
             var expected = _context.Posts.Where(x => x.PostDate > utcNow).ToArray();
 
             //Act
-            var test = _context.Posts.Where("PostDate > DateTime(\"" + utcNow + "\")").ToDynamicArray();
+            var test = _context.Posts.Where("PostDate > DateTime(\"" + utcNow.ToString("u") + "\")").ToDynamicArray();
 
             //Assert
-            CollectionAssert.AreEqual(expected, test);
+            CollectionAssert.AreEqual(expected, test, string.Format("expected: {0} found: {1}", expected.Length, test.Length));
```

Comment:
If I am not mistaken the CollectionAssert automatically writes the error message regarding count mismatch. Why do we need the message here?

Reply:
I wasn't able to confirm the need to fix DateTime tests (I wasn't able to reproduce it nether on my machine nether in Azure Dev Ops).